doc/articles/common-issues-vs2022.md

```diff
@@ -47,6 +47,14 @@ Android deployment requires a few considerations:
         1. In the android section search for `Fast Deployment`
         1. uncheck all target platforms
 
+### My application does not start under WSL
+
+Your application may fail to run under WSL for multiple reasons:
```

Comment:
@jeromelaban I think it would be helpful to share the actual error I am getting when using spaces or special characters so it is easier to find when researching in the doc or on the repo (even for us as a future reference).

![image](https://github.com/user-attachments/assets/df70509f-179b-4155-8341-f956ffc7dfaa)
`The target process exited without raising a CoreCLR started event. Ensure that the target process is configured to use .NET Core. This may be expected if the target process did not run on .NET Core.`

Reply:
We can't know that part at this time, as we do not know whether WSL is being used from VS.